config/locales/en.yml

```diff
@@ -30,4 +30,59 @@
 # available at http://guides.rubyonrails.org/i18n.html.
 
 en:
-  hello: "Hello world"
+  activerecord:
```

Comment:
Forgot all about I18n!!! @bradf83, you must get a rash when I do this!

Reply:
Nah I understand the value of I18n.  But in all the applications I have ever worked on I have only ever used English, and I have implemented I18n in quite a few to have it go to waste.  The one day it is used I will either be bitten because I don't have it or thankful I do.